### PR TITLE
Update mcr.cpp 1st

### PR DESCRIPTION
**Pull request by Chen-da-x**

### CODE REVIEW

**Comment by Chen-da-x** [comment]
This is simple test for pulling new request

File: mcr.cpp [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
 bool win = false; 
-    if (game[0][0] == game[0][1] && game[0][1] == game[0][2] && (game[0][0] == 'X' || game[0][0] == 'O')) win = true; 
-    if (game[1][0] == game[1][1] && game[1][1] == game[1][2] && (game[1][0] == 'X' || game[1][0] == 'O')) win = true; 
-    if (game[2][0] == game[2][1] && game[2][1] == game[2][2] && (game[2][0] == 'X' || game[2][0] == 'O')) win = true; 
+    if (game[0][0] == game[0][1] && game[0][1] == game[0][2] && (game[0][0] == 'X' || game[0][0] == 'O')) 
+        win = true; 
+    if (game[1][0] == game[1][1] && game[1][1] == game[1][2] && (game[1][0] == 'X' || game[1][0] == 'O')) 
+        win = true; 
+    if (game[2][0] == game[2][1] && game[2][1] == game[2][2] && (game[2][0] == 'X' || game[2][0] == 'O')) 
+        win = true;
```
Comment (BeccaYe) [inline mcr.cpp:8]: Reviewed

**Review by BeccaYe — COMMENTED** [review]
Modified format

File: mcr.cpp (hunk 1 above)
Comment (Chen-da-x) [reply mcr.cpp:8]: Ok

**Review by BeccaYe — COMMENTED** [review]
Incorrect input. The array subscripts are started at 0, thus the input should start from 0 rather than 1.

**Pull request merged**